static/app/utils/analytics/dashboardsAnalyticsEvents.tsx

```diff
@@ -45,9 +45,15 @@ export type DashboardsEventParameters = {
   'dashboards_manage.change_sort': {
     sort: string;
   };
+  'dashboards_manage.change_view_type': {
+    view_type: DashboardsLayout;
+  };
+  // TODO(nikki): may not need this one
+  'dashboards_manage.count_favourites': {count: number; user_id: string};
   'dashboards_manage.create.start': {};
   'dashboards_manage.delete': {dashboard_id: number; view_type: DashboardsLayout};
   'dashboards_manage.duplicate': {dashboard_id: number; view_type: DashboardsLayout};
+  'dashboards_manage.favourite': {dashboard_id: string; favourited: boolean};
```

Comment:
```suggestion
  'dashboards_manage.favorite': {dashboard_id: string; favorited: boolean};
```
🇨🇦 👮🏻

Reply:
Might be confusing that an "unfavorite" event has the same name! Maybe `toggle_favorite` would be better?